### PR TITLE
Update change log

### PR DESCRIPTION
**Pull request by gcentauri**
Summarizing: https://github.com/syl20bnr/spacemacs/commits/develop?before=69a89cc41243cfb189238a2297fde4498d7fcf6d+998
Starting With: 43b35d5f0599f1bce6f17f09a6575822f34ea490

New pointer: https://github.com/syl20bnr/spacemacs/commits/develop?before=69a89cc41243cfb189238a2297fde4498d7fcf6d+963
Start with: 4fa20328a6de69393f8bbc4e5e90abbad781b965

### CODE REVIEW

**Comment by gcentauri** [comment]
I bundled the commits related to the new mode-line customization into the Dotfile Changes section, and several commits related to package loading into Core changes.  I hope i made sense of it correctly.

File: CHANGELOG.develop [hunk 1]
```diff
@@ -639,6 +660,8 @@ is enabled
   Rich Alesi)
 - Set =ivy-use-selectable-prompt= to =t= (thanks to Boris Buliga)
 - Rename git dir variable to upstream changes (thanks to bmag)
+- Add =ivy-enable-advanved-buffer-information= variable to toggle =ivy-rich=
+being enabled (thanks to Alejandro Catalina)
```
Comment (Miciah) [inline CHANGELOG.develop]: Typo: "advanved".  Also, the indentation is inconsistent with the rest of the file—when wrapping a line with a list item, the continuation should be indented.

Comment (randomwangran) [reply CHANGELOG.develop]: yes! the "b" in being should inline with "A" Add in line 663 :)

**Comment by sdwolfz** [comment]
Thank you :heart:! 
And congratulations on your first Spacemacs PR :tada:!
Cherry-picked into `release_notes_0.300` branch, you can safely delete your branch.
Commit: 6a0f0826b98ab190e4ce35d4705aabc4588753d6

**Pull request closed without merging**